lua/rip-substitute/config.lua

```diff
@@ -20,14 +20,23 @@ local defaultConfig = {
 	keymaps = {
 		-- normal & visual mode
 		confirm = "<CR>",
+		confirmAll = "<S-CR>",
```

Comment:
for the sake of not introducing a breaking change for existing users, the keymaps should be switched.

`confirm` should be used to confirm all, and your feature should introduce a `confirmSingle` which uses a new keymaps `<S-CR>`. Yes, it's not ergonomic, but otherwise your PR creates a breaking change which will irrirate users.